examples/spark/pyspark_udfs/pandas_udfs.py

```diff
@@ -0,0 +1,32 @@
+"""
+Pandas UDFs.
+Has to only contain map operations!
+"""
+import pandas as pd
+
+from hamilton.function_modifiers import tag
+
+
+@tag(return_type="float")
```

Comment:
let's have a special tag (namespaced0 for this. That way, if we want to make a decorator we can do so.

Actually, a simple decorator wrapping `tag` or `NodeDecorator` might be nice here... We have the ability to have "internal" tags that no one else can use...